Jenkinsfile

```diff
@@ -6,11 +6,20 @@ def getBasePath = { String filepath ->
 	return filepath.substring(0, filename)
 }
 
+def make = { String target ->
+	try {
```

Comment:
Curious if this could be rewritten/simplified to:
```
def make = { String target ->
	try {
		sh "make ${target}"
	} catch(error) {
		false
	}
	sh "make fileperms"
}
```
... or would that last `sh` invocation not get run?